src/main/java/inf112/app/objects/Robot.java

```diff
@@ -229,6 +237,23 @@ public void addDamageTokens(int dealDamage) {
         System.out.println("Damage tokens:" + damageTokens);
     }
 
+    /**
+     * method for deaing the right amount of cards compared to damageTokens
+     */
+    public void dealNewCards() {
+        int amount = 9;
+        if (powerDown){
+            ArrayList<ICard> playerDeck = new CardDeck().getCards(0);
```

Comment:
Det her er farlig, for nå har vi plutselig flere kortstokker rundt omkring

Reply:
Må ha kortstokk (sidecardslots fra cardUI) i Robot heller enn der, også kan den heller hente fra robot